### ISSUE TITLE
Add new entry to Common Code Table C-8 for Metop Second Generation (Metop-SG) instruments

### ISSUE THREAD

**Issue by SimonElliottEUM** [wmo-im/CCT#70]
# Branch

\[add when created\]

# Summary and purpose

Data from the nine instruments on the Metop Second Generation (Metop-SG) satellites will be converted into BUFR for international exchange. Suitable entries for the instruments are needed.

# Action proposed

Allocate entries 210 - 213 in Common Code Table C-8 as per detailed proposal

# Discussions

Standard request for new entry in common code table

# Detailed proposal

The entry should be as follows ...

Code: 230
Agency: EUMETSAT - ESA
Type: Radiometer
Short name: 3MI
Long name: Multi-Viewing Multi-Channel Multi-Polarisation Imaging instrument

Code: 231
Agency: EUMETSAT-CNES
Type: Sounder - interferometer
Short name: IASI-NG
Long name: Infrared Atmospheric Sounding Interferometer - New Generation

Code: 232
Agency: EUMETSAT - DLR
Type: Radiometer
Short name: METimage
Long name: Meteorological Imager

Code: 233
Agency: EUMETSAT - ESA
Type: Radiometer
Short name: MWS
Long name: Microwave Sounder

Code: 234
Agency: EUMETSAT - ESA
Type: Radio Occultation
Short name: RO
Long name: Radio Occultation

Code: 235
Agency: EUMETSAT - ESA - EU
Type: Spectrometer
Short name: S5/UVNS
Long name: Copernicus Sentinel-5 Ultraviolet Visible Near-infrared and Shortwave-infrared Spectrometer

Code: 236
Agency: EUMETSAT - ESA
Type: Radiometer
Short name: ICI
Long name: Ice Cloud Imager

Code: 237
Agency: EUMETSAT - ESA
Type: Radiometer
Short name: MWI
Long name: Microwave Imager

Code: 238
Agency: EUMETSAT - ESA
Type: Radar
Short name: SCA
Long name: Scatterometer

**Comment by SimonElliottEUM** [comment]
@amilan17 I made a small typographic error in the Action Proposed section (thanks for spotting that @sebvi ). The text should refer to "entries 230 - 238"

**Comment by SimonElliottEUM** [comment]
@jitsukoh @amilan17 This branch is ready for the next step

**Comment by amilan17** [comment]
@SimonElliottEUM please update the branch

**Comment by SimonElliottEUM** [comment]
@jitsukoh @amilan17 This branch is now really ready for the next step. I have updated the CSV file accordingly

**Comment by jitsukoh** [comment]
@SimonElliottEUM thank you for the update.
@amilan17 I confirm that the branch is updated and the issue is moved to the validated status.

**Comment by amilan17** [comment]
It's ready to merge. @SimonElliottEUM, please note that I removed "instrument" from the 3MI long name.

**Comment by amilan17** [comment]
@SimonElliottEUM @jitsukoh 

For editorial consistency, I changed the agency names to be separated with "/" instead of " - " -- please review this diff asap. 

e9587ff

**Comment by jitsukoh** [comment]
@amilan17 thank you for the update. "/" seems like "or" but if it is the existing practice, I would agree to make them consistent.

**Comment by SimonElliottEUM** [comment]
@amilan17 @jitsukoh I agree with Jitsuko. Although " - " might seem clearer, we are better off staying with the existing practice and using "/".
